### PR TITLE
Add parts on reader and specific examples

### PR DESCRIPTION
**Pull request by LaureRC**
First draft to add a part on Reader + specific examples on how to call methods in parallel in different cases

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -737,3 +739,103 @@ pipe(
...
+}): ReaderTaskEither<Deps, Error, Value> =>
+  pipe(
+    getById(firstId),
+    ReaderTaskEither.chain(() => sendEvent({ secondId, eventData }))
+  );
+
+// with:
+const getById = (firstId) =>
+  pipe(
+    Reader.ask<Deps>(),
```
Comment (bastoche) [inline README.md:780]: what is `Deps` here?
I remember asking myself this question when I started learning about Reader

**Review by bastoche — APPROVED** [review]
nice!

File: README.md [hunk 2]
```diff
@@ -737,3 +739,103 @@ pipe(
...
+```typescript
+import * as RTE from "fp-ts/ReaderTaskEither";
+import { pipe } from "fp-ts/function";
+
+declare const foo: RTE.ReaderTaskEither<R1, E1, A1>;
+declare const bar: RTE.ReaderTaskEither<R2, E2, A2>;
+
+pipe(
+  RTE.Do,
+  RTE.bindW("foo", () => foo),
```
Comment (vinassefranche) [inline README.md:809]:
```suggestion
  RTE.apS("foo", foo),
```
It does not change much, as far as I know, but I find it clearer like this.
anyway, you can get rid of the `W` if you keep the `bind`

File: README.md [hunk 3]
```diff
@@ -737,3 +739,103 @@ pipe(
...
+  RTE.apSW("bar", bar)
+  // this returns RTE.ReaderTaskEither<R1 & R2, E1 | E2, {readonly foo: A1; readonly bar: A2}>
+);
+```
+
+In the specific case of two `TaskEither` having the same error but not the same result, you can also do the following :
```
Comment (vinassefranche) [inline README.md:815]: I'm wondering if these two examples below are useful. I would say it's better to stay consistent and use `apS` for `TaskEither` too. WDYT?

File: README.md (hunk 1 above)
Comment (LaureRC) [reply README.md:780]: Oh right, I forgot to define what it is in this case, I'll add it now!

File: README.md (hunk 3 above)
Comment (LaureRC) [reply README.md:815]: Well actually yes I agree with you. I wanted to show all the possible ways of doing it that I discovered during the feature we did together yesterday. I don't really have arguments against using `apS` for those too. So let's be consistent as you suggested :) I'll remove those 2 other examples!

File: README.md (hunk 1 above)
Comment (LaureRC) [reply README.md]: I've updated my PR to try to explicit what was `Deps`, is it better now?

**Review by vinassefranche — APPROVED** [review]

**Pull request merged**